reference-tests/src/test/java/net/consensys/linea/BlockchainReferenceTestTools.java

```diff
@@ -194,6 +202,9 @@ public static void executeTest(final BlockchainReferenceTestCaseSpec spec) {
     zkTracer.traceStartConflation(spec.getCandidateBlocks().length);
 
     for (var candidateBlock : spec.getCandidateBlocks()) {
+      Assumptions.assumeTrue(
+          candidateBlock.areAllTransactionsValid(),
```

Comment:
I don't fully understand this.  Does `areAllTransactionsValid()` indicate only that transactions must be well formed?  Presumably a transaction which reverts because of an out-of-gas condition would not be considered "invalid"?